mathics/builtin/box/expression.py

```diff
@@ -1,6 +1,8 @@
 # This is never intended to go in Mathics3 docs
 no_doc = True
```

Comment:
BTW current linters want  `no_doc = True`  after all imports.  But if you don't want to address this here it can be done in a future PR.

Reply:
Yeah, let's leave that for another one.